src/app-layer.c

```diff
@@ -690,6 +690,7 @@ static int TCPProtoDetect(ThreadVars *tv, TcpReassemblyThreadCtx *ra_ctx,
 parser_error:
     ExceptionPolicyApply(p, g_applayerparser_error_policy, PKT_DROP_REASON_APPLAYER_ERROR);
     AppLayerIncrErrorExcPolicyCounter(tv, f, g_applayerparser_error_policy);
+    AppLayerIncFlowCounter(tv, f);
```

Comment:
Again, this path is reachable to count again a flow which has been counted already like /* account flow if we have both sides */

Reply:
I did think about this one and was worried about missing any other call paths that may lead to double increment but then QA stats gave me false confidence. 😅